codecov.yml

```diff
@@ -1,2 +1,2 @@
 fixes:
-  - "::src/"
+  - "/home/runner/work/label-studio-frontend/label-studio-frontend/::"
```

Comment:
absolute path is not backward compatible with local run

Reply:
we don't care about local runs as we're not uploading anything to codecov from local machines. this is a CI only thing so we're not expecting local paths there